src/oc_erchef/apps/data_collector/rebar.lock

```diff
@@ -1,23 +1,22 @@
-[
- %% these apps are not present in the rebar.conf so
- %% commenting out for now to see what happens.
-%{<<"goldrush">>,
-% {git,"git://github.com/DeadZen/goldrush.git",
-%      {ref,"57e71f2789632a085f0963e1244644fdcb1941b9"}},
-% 1},
-%{<<"ibrowse">>,
-% {git,"git://github.com/chef/ibrowse.git",
-%      {ref,"7b2b0bfd5e15e57706beb2ef0360cdc7f24b37c5"}},
-% 1},
+{"1.1.0",
+[{<<"goldrush">>,{pkg,<<"goldrush">>,<<"0.1.9">>},1},
+ {<<"ibrowse">>,
+  {git,"https://github.com/cmullaparthi/ibrowse.git",
+       {ref,"c97136cfb61fcc6f39d4e7da47372a64f7fca04e"}},
+  1},
  {<<"lager">>,
-  {git,"git://github.com/basho/lager.git",
-       {ref,"81eaef0ce98fdbf64ab95665e3bc2ec4b24c7dac"}},
+  {git,"git://github.com/erlang-lager/lager.git",
+       {ref,"f26a8ea0f4c2ed6506a54e0b9890edd8148b9500"}},
```

Comment:
we should make sure there are no related omnibus changes required in any of the sys.config templates.

Reply:
I checked the sys.config files.  All I saw were settings for things like log file size.  Was there anything in particular you wanted me to look into?